allennlp/nn/util.py

```diff
@@ -1307,7 +1306,7 @@ def masked_index_fill(
     # Shape: (nonzero_indices, 1)
     unmasked_indices = flattened_indices[mask].unsqueeze(-1)
 
-    flattened_target.scatter_(0, unmasked_indices, fill_value)
+    flattened_target.scatter(0, unmasked_indices, fill_value)
```

Comment:
I don't think that works. It has to be
```suggestion
    flattened_target = flattened_target.scatter(0, unmasked_indices, fill_value)
```
I'm hoping that this fails tests, because if it passes something is wrong with the tests ...

Reply:
haha ok they failed.